apps/demos/Demos/DropDownBox/SingleSelection/Angular/app/app.component.ts

```diff
@@ -69,21 +70,33 @@ export class AppComponent {
   }
 
   treeView_itemSelectionChanged(e: DxTreeViewTypes.ItemSelectionChangedEvent) {
-    this.treeBoxValue = e.component.getSelectedNodeKeys();
+    const selectedKeys = e.component.getSelectedNodeKeys();
+    this.treeBoxValue = selectedKeys.length > 0 ? selectedKeys[0] : null;
+  }
+
+  treeView_itemClick() {
```

Comment:
why do we need additionally onItemClick to close an editor. 
ideally everything should be done within a single event , e,g, `treeView_itemSelectionChanged` ?